### DISCUSSION TITLE
Is anyone successfully using Facebook login in either v18 or v20?

### DISCUSSION THREAD

**Discussion by vicos59** [geodesicsolutions-community/geocore-community#186] [Q&A]
Curious to see if anyone else is successfully using Facebook login with either v18 or v20? I can't get it to work. 

If it does not work for anyone, I'll assume that the code is out of date (Facebook pushes the standards frequently).

Is anyone interested in sharing the cost to implement a multi-platform social login like Google Firebase:

https://firebase.google.com/products/auth

https://youtu.be/8sGY55yxicA

**Comment by jonyo** [comment]
For context, they did get it working a year or 2 back (some time after v18 came out) and I believe those changes are in the current v20 code.  But I would not be surprised if FB has since made further changes that break it again like you mention. 🙄

**Reply by vicos59** [reply]
I suspect Facebook has changed the API again as within the past year or so I had to update Xenforo to keep it working. Received the same error when Xenforo broke as I am now with GeoCore. 

Anyway, hope to get some confirmation from other users. I'll push this convo to Facebook page.

**Reply by vicos59** [reply]
@jonyo Do you know if bad things happen if you disable the Facebook login addon when you have users who initially signed up using Facebook. I have several hundred users like this and I need to disable it since it no longer works.

I know there is a button to unlink them from Facebook. I assume that will go away, but if so, do they become a 'hung' user?

**Reply by jonyo** [reply]
It's been a while so you will need to confirm this by testing, but if I recall correctly they will be able to use "forgot password" to get a password set then they can log in.
